esx_service/vmdk_ops.py

```diff
@@ -539,10 +546,13 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     Returns None (if all OK) or error string
     """
-
     vm_datastore = get_datastore_name(config_path)
+    result, tenant_uuid, tenant_name = auth.authorize(vm_uuid, vm_datastore, cmd, opts)
+    if result:
```

Comment:
would be better (and cleaner IMO) to rename "result" into "error_info" or something like that.